app/controllers/films_controller.rb

```diff
@@ -7,10 +7,12 @@ class FilmsController < ApplicationController
   decorates_assigned :film
 
   def index
-    @films = Film.all
+    @films = Film.all.decorate
```

Comment:
Draper has option decorates_assigned for collections - check it in documentation